lms/djangoapps/gating/signals.py

```diff
@@ -21,10 +19,13 @@ def handle_score_changed(**kwargs):
     Returns:
         None
     """
-    course = modulestore().get_course(CourseKey.from_string(kwargs.get('course_id')))
+    course = kwargs['course']
     if course.enable_subsection_gating:
+        subsection_grade = kwargs['subsection_grade']
+        new_score = subsection_grade.graded_total.earned / subsection_grade.graded_total.possible * 100.0
```

Comment:
It feels strange to me to still have to do this arithmetic here. I know it's not in the scope of this ticket, but it seems like maybe the gating API should take `new_score` as a decimal between 0 and 1 instead of an int between 0 and 100.